etp-backend/src/main/clj/solita/etp/api/valvonta_oikeellisuus.clj

```diff
@@ -103,4 +104,15 @@
                             (api-response/put-response
                               (valvonta-service/update-toimenpide!
                                 db whoami id toimenpide-id body)
-                              (str "Toimenpide " id "/" toimenpide-id " does not exists.")))}}]]]]])
+                              (str "Toimenpide " id "/" toimenpide-id " does not exists.")))}}]
+       ["/publish"
```

Comment:
Tästä saisi varmaan REST-tyylisen siten, että published olisi toimenpiteen kenttänä. Tämä on kyllä näin vähintään yhtä hyvä ellei parempi, joten ei kannata muuttaa.